cmd/trade.go

```diff
@@ -67,6 +64,27 @@ func init() {
 			log.Println()
 			log.Fatal(e)
 		}
+
+		if *logPrefix != "" {
+			//t := time.Now().String()
```

Comment:
Can you remove the commented out code and sample string?

Reply:
Yep, meant to do that.